### ISSUE TITLE
[css‑shadow‑parts] What is the specificity of `::part(…)` with multiple `<ident>`s?

### ISSUE THREAD

**Issue by ExE-Boss** [w3c/csswg-drafts#3995]
https://drafts.csswg.org/css-shadow-parts/#part

Since the part name is similar to a class name, I would expect that `::part(active tab)` would have a higher specificity than `::part(tab)` or `::part(active)`.

**Comment by tabatkins** [comment]
Sounds reasonable, but do we want this to interact class-like with real classes?

That is, do the following have the same specificity, so the result is solely determined by order?

```
.foo::part(tab) {color: blue;}
::part(active tab) {color: red;}
```

**Comment by emilio** [comment]
I think I'd rather keep it working as a regular pseudo-element selector than complicating the specificity rules, but that's just my opinion :)

**Comment by ExE-Boss** [comment]
> Sounds reasonable, but do we want this to interact class-like with real classes?
> 
> That is, do the following have the same specificity, so the result is solely determined by order?
> 
> ```css
> .foo::part(tab) {color: blue;}
> ::part(active tab) {color: red;}
> ```

Well, `.foo` selects the shadow host in the light tree, whereas `::part(active tab)` selects the part in the shadow tree, so ¯\\\_(ツ)\_/¯.

**Comment by emilio** [comment]
I don't think that matters? Both can end up matching the same element, so they need to be sorted by specificity and thus tab's question still stands.
